developers/requesting-data-commitment-ranges.md

```diff
@@ -0,0 +1,26 @@
+# Requesting data commitment ranges
+
+To request proofs to be submitted to the Blobstream X contract at a
```

Comment:
start this with "By default, the Succinct team will be maintaining canonical Blobstream X deployments on ... chains, updating every X hours. If you wish for the Blobstream X contract to be updated at a different cadence, then you have several different options for how to update the smart contract."

Reply:
which chains should I list here?

Reply:
https://github.com/celestiaorg/docs/pull/1379/commits/0178c3ac5a52b09e1fdeec8a45480ae54e1f0ef7